decentraland/components/presentational/TinyLink.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { StyleSheet, Text } from "react-native";
+import {
+  responsiveHeight,
+  responsiveFontSize,
+} from "react-native-responsive-dimensions";
+import Colors from "@constants/Colors";
+import PropTypes from "prop-types";
+import { Button } from "native-base";
+import { openURL, showError } from "@helpers";
+
+const openLink = async url => {
+  try {
+    await openURL(url);
+  } catch (err) {
+    showError(`Unable to open link`);
+  }
+};
+
+export default function TinyLink({ text, url }) {
+  const onPress = () => {
+    openLink(url);
+  };
+  return (
+    <Button transparent onPress={onPress}>
```

Comment:
I think in this instance I might prefer a clickable view with just text to a button. Do you think the button makes it more tempting to click than it actually should be for the average user?

Reply:
Yes, I think. I'll do that change.

Reply:
Oh, I guess that is how it looks on iOS

![Simulator Screen Shot - iPhone Xʀ - 2019-04-10 at 17 21 29](https://user-images.githubusercontent.com/2731712/55917523-38b36580-5bb5-11e9-8a53-ea43cd039ad0.png)

Reply:
![t](https://user-images.githubusercontent.com/29674218/55917591-c4de8080-5be7-11e9-97ff-2204683094e5.jpeg)

Reply:
![b](https://user-images.githubusercontent.com/29674218/55917597-ca3bcb00-5be7-11e9-9a52-450121ce470d.jpeg)

Reply:
Before and after change

Reply:
Oh, sorry I've read again and seen that you said `clickable view`. Is the link working on iOS?

Reply:
@marcelomorgado Wait, which of those is before and which is after?

Reply:
> Oh, sorry I've read again and seen that you said `clickable view`. Is the link working on iOS?

I'll check and see if the link is working on iOS

Reply:
> @marcelomorgado Wait, which of those is before and which is after?

Here the transparency is working, the first image is how it was on this PR. The second image was the same but without transparency (I've misunderstood what you have asked).

Reply:
> > Oh, sorry I've read again and seen that you said `clickable view`. Is the link working on iOS?
> 
> I'll check and see if the link is working on iOS

Yep, the link feature was working well as of the commit I originally reviewed

Reply:
So, this PR is ready to merge the way that it is?

Reply:
This is the way we don't want it to look. As of commit https://github.com/tasitlabs/tasit/pull/279/commits/64a576327e7d36ac01c5bd2d43d978f106dd652b, it appears that it doesn't look this way on either Android or iOS (I tested both locally), so that's good. You can disregard my originally comment, in that case.

![55917597-ca3bcb00-5be7-11e9-9a52-450121ce470d](https://user-images.githubusercontent.com/2731712/55918260-15d68080-5bb8-11e9-904f-e011b83ca4e2.jpeg)

Reply:
> > @marcelomorgado Wait, which of those is before and which is after?
> 
> Here the transparency is working, the first image is how it was on this PR. The second image was the same but without transparency (I've misunderstood what you have asked).

Ah ok, good.

Reply:
> So, is this PR ready to merge the way that it is?

Yep!